frontend/packages/console-app/src/components/quick-starts/catalog/QuickStartTileHeader.tsx

```diff
@@ -29,14 +29,17 @@ const QuickStartTileHeader: React.FC<QuickStartTileHeaderProps> = ({ status, dur
 
   return (
     <div className="co-quick-start-tile-header">
-      <Title headingLevel="h3">{name}</Title>
+      <Title headingLevel="h3" data-test={`tile-${name}`}>
```

Comment:
```suggestion
      <Title headingLevel="h3" data-test="title">
```

Similar to above, this should be just `title`.

To differentiate the different tiles you should add a data-test={`tile ${name}`} to `QuickStartTile.tsx`.

I would prefer here if we use the QuickStart resource name here instead of the displayName?!

In `QuickStartTile.tsx` I would suggest:

```diff
   const {
+    metadata: { name },
     spec: { icon, displayName, description, durationMinutes, prerequisites },
   } = quickStart;

  ...

   return (
     <CatalogTile
       icon={quickStartIcon}
       className="co-quick-start-tile"
       featured={isActive}
       title={<QuickStartTileHeader name={displayName} status={status} duration={durationMinutes} />}
       onClick={onClick}
+      data-test={`tile ${name}`}
     >
       <QuickStartTileDescription description={description} prerequisites={prerequisites} />
     </CatalogTile>
```

You need update your page object to

```tsx
  installOpenshiftPipelineOperatorCard: '[data-test~="tile"][data-test~="explore-pipelines"]',
  getStartedWithASampleApplicationCard: '[data-test~="tile"][data-test~="sample-application"]',
```

Reply:
Thanks for the suggestion, updated the code